pyserini/prebuilt_index_info.py

```diff
@@ -1887,6 +1887,398 @@
         "downloaded": False
     },
 
+    # BEIR (v1.0.0) contriever indexes
+    "beir-v1.0.0-trec-covid-contriever": {
```

Comment:
```suggestion
    "beir-v1.0.0-trec-covid.contriever": {
```

Let's change to above convention of `beir-v1.0.0-trec-covid.{model}`?

Reply:
changed `-` to `.`